diesel_migrations/migrations_internals/src/migration.rs

```diff
@@ -12,6 +13,33 @@ pub trait Migration {
     }
 }
 
+#[allow(missing_debug_implementations)]
+#[derive(Clone, Copy)]
```

Comment:
Oh I wonder if the lint only triggers if the type has a clone impl. That would make sense.